### PR TITLE
MayankRaje-fix-issue-2587: Make the account cards smaller

### LINKED ISSUE #2587: [FEATURE] Smaller account cards and multi-line account selection
### Please confirm the following

- [X] I checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate issues

### What do you want to be added or improved?

The account cards are quite large and hence it's diffcult to glance through all the accounts. Especially, when one has 6-7 accounts including banks and cards. So, I request to make them smaller to fit all in a one screen.

Also, while adding expense/income, account is shown in a single line. Hence, need to scroll a long to get desired account when one has multiple bank accounts and cards. I request to make this account selection to multi-line mode, like:

```
Banks: Bank-A | Bank-B | Bank-C
Cards: Card-X | Card-Y | Card-Z
Wallet: Wallet-M | Wallet-N | Wallet-O
Cash
```

Thanks :-)

### Why do you need it?

- 

### How do you imagine it?

_No response_

### PR DESCRIPTION
**Pull request by MayankRaje**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [x] The PR is submitted to the `main` branch.
- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the "Contributing Rules".
- [x] I've read the [Architecture Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Architecture.md).
- [x] I confirm that I've run the code locally and everything works as expected.
- [x] 🎬 I've attached a **screen recoding** of the changes. 

> Tip: drag & drop the video to the PR description.

## What's changed?

Describe with a few bullets **what's new:**
- a The account cards are smaller in size.
- b
- c
- d

> 💡 Tip: Please, attach screenshots and screen recordings. It helps a lot!

## Risk Factors

**What may go wrong if we merge your PR?**

- a nothing
- b
- c

**In what cases your code won't work?**

- a it won't happen
- b
- c

## Does this PR closes any GitHub Issues?

Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.

- Closes #2587

> Replace `ISSUE_NUMBER` with your issue number (for example Closes #1234). If you've done that correctly, you'll see the issue title linked when previewing your PR description.

## Troubleshooting CI failures

If you see any of the PR checks failing (❌) go to [Actions](https://github.com/Ivy-Apps/ivy-wallet/actions) and find it there. Or simply click "Details" next to the failed check and explore the logs to see why it has failed.

### Detekt
[Detekt](https://detekt.dev/) is a static code analyzer for Kotlin that we use to enforce code readibility and good practices.

**To run Detekt locally:**
```
./gradlew detekt
```

If the Detekt errors are caused by a legacy code, you can suppress them using a basline.

**Detekt baseline** (not recommended)
```
./scripts/detektBaseline.sh
```

### Lint

We use the [standard Android Lint](https://developer.android.com/studio/write/lint) plus [Slack's compose-lints](https://slackhq.github.io/compose-lints/) as an addition to enforce proper Compose usage.

**To run Lint locally:**
```
./scripts/lint.sh
```

If the Lint errors are caused by a legacy code, you can suppress them using a basline.

**Lint baseline** (not recommended)
```
./scripts/lintBaseline.sh
```

### Unit tests

If this job is failing this means that your changes break an existing unit test. You must identify the failing tests and fix your code.

**To run the Unit tests locally:**
```
./gradlew testDebugUnitTest
```

### CODE REVIEW

**Review by ILIYANGERMANOV — COMMENTED** [review]
The code looks good, can you upload a screenshots + screen recording to see how it looks visually

**Comment by MayankRaje** [comment]
**Before image:** 
WhatsApp Image 2023-10-14 at 02 03 36


**After image**
WhatsApp Image 2023-10-14 at 02 03 36 (1)

File: screen-accounts/src/main/java/com/ivy/accounts/AccountsTab.kt [hunk 1]
```diff
@@ -279,16 +279,16 @@ private fun AccountHeader(
...
 
         BalanceRow(
             modifier = Modifier
                 .align(Alignment.CenterHorizontally)
                 .clickableNoIndication {
                     onBalanceClick()
                 },
-            decimalPaddingTop = 7.dp,
-            spacerDecimal = 6.dp,
+            decimalPaddingTop = 1.dp,
```
Comment (ILIYANGERMANOV) [inline screen-accounts/src/main/java/com/ivy/accounts/AccountsTab.kt:290]: Increase the decimalPaddingTop because the 0.00 looks weird

**Comment by MayankRaje** [comment]
**Video:**
__________


https://github.com/Ivy-Apps/ivy-wallet/assets/71680976/be49a751-03f9-4dc7-92c1-6b52aeb48e62

**Review by ILIYANGERMANOV — APPROVED** [review]
Looks good, just fix the 0.00 so the .00 looks like it was before

File: screen-accounts/src/main/java/com/ivy/accounts/AccountsTab.kt (hunk 1 above)
Comment (MayankRaje) [reply screen-accounts/src/main/java/com/ivy/accounts/AccountsTab.kt]: **Increased the decimalPaddingTop to 7.dp**
__________________________________________________

WhatsApp Image 2023-10-14 at 14 56 13

**Review by ILIYANGERMANOV — APPROVED** [review]
Thanks for fixing it! 🎉

**Pull request merged**